assets/less/bootstrap/modals.less

```diff
@@ -25,9 +25,8 @@
   position: fixed;
   top: 10%;
   left: 50%;
+  transform: translateX(-50%);
```

Comment:
I'd rather see these overrides in `fauxton.less`. There's already a section with some in there (search for `bootstrap overrides`). 
I was able to make it work by adding this to `fauxton.less`:
```
.modal {
  transform: translateX(-50%);
  margin-left: inherit;
}
.modal-dialog {
  display: flex;
  flex-direction: row;
}
 .modal-content {
  flex: 1;
  width: 560px;
}
```